README.md

```diff
@@ -1,2 +1,20 @@
 # sg-agent
-Availability monitoring agent for SG
+
+## Set of sg-core application/transport plugins for spawning availability monitoring agent STF
+
+sg-agent serves as availability monitoring agent for STF. It executes
+self-scheduled and/or sg-core requested checks and reports results back.
+
+As a matter of fact sg-agent will not need sg-core as a server side component.
+The plan for sg-agent is to be self sufficient without server side or cooperate
+well with other availability monitoring solutions such as sensu-core, sensu-go,
+Nagios, Icinga, Zabbix or even Datadog.
+
+TO-DO:
```

Comment:
Looks to me like there is some markdown issue here. Look at the rendered readme: https://github.com/paramite/sg-agent/blob/sft-519-scheduler/README.md
It's missing newlines between the checkboxes